src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
 
-The per-task metrics are displayed first, starting with the name of our Task Set, `LoadtestTasks`. Individual tasks in the Task Set are then listed in the order they are defined in our load test. We did not name our task, so it simply shows up as "1: ". All defined tasks will be listed here, even if they did not run, so this can be useful to confirm everything in your load test is running as expected.
+![HTML report request metrics section](report_request_metrics.png)
+
+The per-request metrics are displayed first. Our single task makes a `GET` request for the `/` path, so it shows up in the metrics as `GET /`. The table in this section displays shows the total number of requests made (2,499),the average number of requests per second (208.25), and the average number of failed requests per second (0). 
+
+Additionally is shows the average time required to load a page (39.68 milliseconds), the minimum time to load a page (19 ms) and the maximum time to load a page (180 ms).
 
-Next comes the per-request metrics. Our single task makes a `GET` request for the `/` path, so it shows up in the metrics as `GET /`. Comparing the per-task metrics collected for `1: ` to the per-request metrics collected for `GET /`, you can see that they are the same.
+![HTML report response times metrics section](report_response_metrics.png)
 
-There are two common tables found in each type of metrics. The first shows the total number of requests made (2,054), how many of those failed (0), the average number of requests per second (410.8), and the average number of failed requests per second (0).
+The second section displays the average time required to load a page. The table in this section is showing the slowest page load time for a range of percentiles. In our example, in the 50% fastest page loads, the slowest page loaded in 36 ms. In the 70% fastest page loads, the slowest page loaded in 41 ms, etc.
 
-The second table shows the average time required to load a page (20.68 milliseconds), the minimum time to load a page (7 ms), the maximum time to load a page (254 ms) and the median time to load a page (19 ms).
+![HTML report tasks metrics section](report_task_metrics.png)
 
-The per-request metrics include a third table, showing the slowest page load time for a range of percentiles. In our example, in the 50% fastest page loads, the slowest page loaded in 19 ms. In the 75% fastest page loads, the slowest page loaded in 21 ms, etc.
+Next comes per-task metrics, starting with the name of our Task Set, `LoadtestTasks`. Individual tasks in the Task Set are then listed in the order they are defined in our load test. We did not name our task, so it simply shows up as `0.0`. All defined tasks will be listed here, even if they did not run, so this can be useful to confirm everything in your load test is running as expected. Comparing the tasks metrics metrics collected for `0.0` to the per-request metrics collected for `GET /`, you can see that they are the same.
```

Comment:
To address the info removed above, perhaps add another sentence, for example:

Comparing the tasks metrics metrics collected for `0.0` to the per-request metrics collected for `GET /`, you can see that they are the same. This is because in our simple example, our single task only makes one request.

Reply:
Done.